lib/ansible/modules/network/nxos/nxos_vlan.py

```diff
@@ -48,7 +48,7 @@
   vlan_state:
     description:
       - Manage the vlan operational state of the VLAN
-        (equivalent to state {active | suspend} command.
+        This is being deprecated in favor of state:
```

Comment:
`I(state)`

http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options

Reply:
Fullstop rather than `:` at the end?